.gitignore

```diff
@@ -81,6 +81,7 @@ celerybeat-schedule
 # virtualenv
 venv/
```

Comment:
Do we want to edit the readme, contributors, or create a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) to suggest that if someone is storing their virtualenv in the project, they choose one of these locations? This way someone doesn't accidentally contribute a PR that includes their virtual environment.